fiftyone/utils/annotations.py

```diff
@@ -1783,6 +1791,15 @@ def supports_keyframes(self):
         """
         raise NotImplementedError("subclass must implement supports_keyframes")
 
+    @property
+    def supports_video_sample_fields(self):
+        """Whether this backend supports annotating video labels at a
+        sample-level or only at a frame-level.
```

Comment:
nit: I'd remove "or only at a frame-level" here.

Maybe in the future we'll have backends that don't support videos at all, in which case this property would be False but frame-level labels aren't supported either (presumably a new `supports_videos=False` parameter would be added)